### ISSUE TITLE
document.xsd?

### ISSUE THREAD

**Issue by bequessel** [bodleian/consolidated-tei-schema#31]
I am using the **msdesc.xsd** from the alternatives folder. Now, when I load this schema into a stylesheet editor (altova style vision) I get an error message that a file **document.xsd** can't be found. I have **msdesc.xsd, xml.xsd, xinclude.xsd** and **dcr.xsd** available in the same folder. **document.xsd** is meant to be imported by **dcr.xsd** wih this line::
```xml
<xs:import namespace="http://www.tei-c.org/ns/1.0" schemaLocation="document.xsd"/>
```
any idea where this might be?

**Comment by ahankinson** [comment]
I wonder if this is a problem with the TEI site being down, so it can't access this XSD.

**Comment by bequessel** [comment]
should it not be in the same folder as msdesc?

**Comment by andrew-morrison** [comment]
_document.xsd_ is the name of the file generated by TEI-C's code which contains the bulk of the rules that define TEI P5 and our customizations. I renamed that to _msdesc.xsd_ to match the other naming convention of the RelaxNG and DTD versions.

But I didn't notice the original filename is referenced in all the other _.xsd_ files, which are libraries for handling other things, like XLink syntax. Presumably it is set up that way so you can validate an TEI document against any of them. Oxygen didn't raise an error when testing, because the namespace in the import statements is already associated with rules, when starting with msdesc.xsd. Trying it now with xmllint the missing _document.xsd_ is noted but validation continues. Altova's parser must be even stricter.

I've changed it so _msdesc.xsd_ is now a stub that does nothing except import _document.xsd_, which then imports the rest. Testing in xmllint returns no warnings. 

@bequessel: Please try it in your software, and re-open this issue if it still causes problems.
